components/divider/stories/divider.stories.js

```diff
@@ -45,5 +47,107 @@ export default {
 	},
 };
 
+/**
+ * The default size for divider is medium.
+ */
 export const Default = Template.bind({});
 Default.args = {};
+Default.parameters = {
+	docs: {
+		story: {
+			height: "auto",
+		}
+	},
+};
+
+const AllDividerSizes = (args) => html`
+	<div>
+		<div style="display: flex; flex-direction: column; padding: 16px;">
+			<h3 style="margin: 0;">Small</h4>
```

Comment:
Could you set the color to white on the headings when Static white is set, like on the docs page? I noticed they were a little hard to read on top of the background.